docker/monitoring/prometheus-remote-write.yaml

```diff
@@ -0,0 +1,41 @@
+global:
+  scrape_interval:     15s
```

Comment:
@kylehuntsman having read a bit more on prometheus over the last week, it is important that we set out ranges to minimum of `1m`, given that our scrape_interval is `15s`. The recommendation for `range` is 4 times the scrape interval, as prometheus needs at least 2 points to do a calculation, and we have to account for missed scrapes.